### DISCUSSION TITLE
Mi dispositivo no me deja instalar termux desde f-droid y necesito la última versión

### DISCUSSION THREAD

**Discussion by RoqueValCor** [termux/termux-app#2353] [Q&A] [answered]
Mi dispositivo no me deja instalar termux desde f-droid Como puedo solucionarlo para contar con la última vesión de termux

**Comment by agnostic-apollo** [comment, marked as answer]
>My device won't let me install termux from f-droid How can I fix it to have the latest version of termux

Uninstall **ALL** termux apps you installed from playstore first. Check apps list in android settings.

https://github.com/termux/termux-app#installation

And we only provide support in English.

**Reply by Aridlvory** [reply]
*only in English?

**Reply by agnostic-apollo** [reply]
`1.` at https://wiki.termux.com/wiki/Community

**Reply by Aridlvory** [reply]
Ohkay...

**Reply by Aridlvory** [reply]
Since GitHub has an inbuilt translator... People can ask in other languages right....

**Reply by Aridlvory** [reply]
But... Answer in English...
